coin-pickup-game/src/index.js

```diff
@@ -23,12 +23,12 @@ const Themes = [
     }
 ]
 
-module.exports = class CoinPickupGame extends BasePlugin {
+export default class CoinPickupGame extends BasePlugin {
 
     /** Plugin info */
-    static get id()             { return 'vatominc-coin-pickup-game' }
-    static get name()           { return 'Coin Pickup Game' }
-    static get description()    { return 'A game where users can walk over coins to collect them.' }
+    static id = "vatominc-coin-pickup-game"
+    static name = "Coin Pickup Game"
+    static description = "A game where users can walk over coins to collect them."
 
     /** Current number of coins collected */
     score = 0
```

Comment:
The autospawn functionality does not work. More specifically:
serverTick is not documented and it does not seem to work. Tried also server_tick_enabled
onTimer() is not documented and it is never invoked.

Reply:
- When `serverTick` is set to `true` in the `registerComponent` method, it will call the `onServerTick()` method every 30 seconds (we do need to document this)
- `onTimer` is not an internal method, it seems to be called right [here](https://github.com/VatomInc/spaces-plugins/blob/master/coin-pickup-game/index.js#L110)

Reply:
Ah you are right on the onTimer :)
Regarding serverTick I read about it here https://developer.vatom.com/plugins/api/objects/ (search for server_tick_enabled) The thing is neither works, I tried with `serverTick: true` and with `server_tick_enabled: true`. I enabled the functionality in the settings but it didn't work. I am probably doing some silly mistake.